decidim-core/app/views/decidim/devise/shared/_boxes.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="login__box-container">
+  <div class="login__box">
```

Comment:
I really like these boxes and the way they look visually but I think we could slightly improve their usability. How about we make the whole box clickable instead of only the tiny link within the box?

Here is e.g. how it looks when I hover over the "Already have an account?" title:
![Devise boxes clickability](https://user-images.githubusercontent.com/864340/186687230-af04e894-58bd-4664-aca8-17d5e5ba7030.png)

Especially on touch devices it can be extremely frustrating when nothing happens after trying to click an element that looks clickable on its own. On desktop, I think I can get it after a couple of misclicks and no hover effects but I think this can be a problem on touch devices.